.github/workflows/instrumentation_tests.yml

```
@@ -1,8 +1,8 @@
 name: Instrumentation tests
 on:
-  schedule:
-    # every 12 hours
-    - cron: '0 */12 * * *'
+  pull_request:
+    paths:
+      - '**'
```

This change should be reverted before committing
```suggestion
  schedule:
    # every 12 hours
    - cron: '0 */12 * * *'
```